test/verify/check-services

```diff
@@ -32,6 +33,15 @@ class TestServices(MachineCase):
         # we don't race with the boot process
         self.machine.execute("systemctl start default.target")
 
+        # We manipulate with `/etc/systemd/system` so `reload-daemon` to keep it in proper state
+        # Also `reset-failed` as failed units can be still listed with `systemctl` even when all files
+        # were removed and daemon reloaded.
+        self.restore_dir("/etc/systemd/system", "systemctl daemon-reload && systemctl reset-failed", True)
+
+        # Many services use this command and when the test finishes/fails while they are active,
+        # `systemct status test.service` still shows this service as active until this process dies
+        self.addCleanup(self.machine.execute, "pkill -fx '/bin/sh -c while true; do sleep 5; done' || true")
```

Comment:
Wouldn't it be more robust to `systemctl stop test.service`? Or are the service names different? Could it do something like `systemctl stop test*.service`?

Reply:
there are `test.service`, `condtest.service`, `test-a.service`, `test-b.service`, test-c.service`.
But those 3 are actually never started. So I can stop those specific services in appropriate tests.